src/bankAccount.js

```diff
@@ -1,7 +1,34 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, sum, birthDate) {
+  const regexp = /^([0-2]\d|3[0,1])\.(0\d|1[0-2])\.(\d{2})$/;
+
+  this.name = name;
+  this.money = sum;
+
+  this.date = birthDate && birthDate.match(regexp)
+    ? birthDate.match(regexp)[0]
+    : undefined;
```

Comment:
это делается не совсем так
выносится в отдельную функцию как минимум

```suggestion
  this.birthDate = birthDate
```
будет достаточно